index.d.ts

```diff
@@ -115,7 +117,11 @@ declare module 'focus-trap' {
      * By default, focus trap on deactivation will return to the element
      * that was focused before activation.
      */
-    setReturnFocus?: FocusTargetValueOrFalse | ((nodeFocusedBeforeActivation: HTMLElement | SVGElement) => FocusTargetValueOrFalse);
+    setReturnFocus?:
+      | FocusTargetValueOrFalse
```

Comment:
Isn't this a syntax error?
```suggestion
      FocusTargetValueOrFalse
```

Reply:
It was a chance from prettier, I will check

Reply:
If you run `yarn test:types` and it passes, then I guess it's fine. Just kind of surprising syntax to me.